packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -74,7 +74,13 @@ export interface VercelServerlessConfig {
 	/** Configuration for [Vercel Web Analytics](https://vercel.com/docs/concepts/analytics). */
 	webAnalytics?: VercelWebAnalyticsConfig;
 
-	/** Configuration for [Vercel Speed Insights](https://vercel.com/docs/concepts/speed-insights). */
+	/**
+	 * @deprecated This option lets you configure the legacy speed insights API which is now deprecated by Vercel.
+	 * 
+	 * See [Vercel Speed Insights Quickstart](https://vercel.com/docs/speed-insights/quickstart) for instructions on how to use the library instead.
+	 * 
+	 * https://vercel.com/docs/speed-insights/quickstart
```

Comment:
Are the duplicated links intentional?

Reply:
I think so, since we expose two modules: one for static and one for serverless

Reply:
They are. To me, it didn't seem obvious that the first one would be clickable so I added the link itself to be safe.